docs/adguard-browser-extension/mv3-version.md

```diff
@@ -0,0 +1,1102 @@
+---
+title: Manifest V3 version
+sidebar_position: 5
+---
+
+As MV2 extensions are being phased out in the Chrome Web Store, Chrome users may need to switch to MV3 extensions that are compatible with Chrome’s new API Manifest V3. Our thoughts on this topic are extensively discussed on [our blog](https://adguard.com/en/blog/tag/manifest-v3.html).
+
+Our MV3 extension effectively blocks ads and trackers while seamlessly managing social widgets, banners, and video ads. Most users will not notice any operational differences, but there are some limitations and changes to be aware of.
+
+[Where to find our extensions](#where-to-find-our-extensions)
+
+[Main changes in features and settings](#main-changes-in-features-and-settings)
+
+[Limitations](#limitations)
+
+[Maximum number of rules and filters](#maximum-number-of-rules-and-filters)
+
+[Network rules](#network-rules)
+
+## Where to find our extensions
+
+The **MV3 version** has replaced our old extension in the [Chrome Web Store](https://chromewebstore.google.com/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg).
+
+The **beta version of MV3** is still available in the [Chrome Web Store](https://chromewebstore.google.com/detail/adguard-adblocker-mv3-exp/apjcbfpjihpedihablmalmbbhjpklbdf).
+
+The old beta extension has been renamed to [**AdGuard Ad Blocker MV2**](https://chromewebstore.google.com/detail/adguard-adblocker-beta/gfggjaccafhcbfogfkogggoepomehbjl) and will be supported until Google phases it out.
+
+## Main changes in features and settings
+
+- **No auto and manual filter updates.** The options *Auto-update filters* and *Check filters update* are no longer available in the *Filters* tab. Since some of the rules are now applied in DNR form, we can’t update filters on request, only through the full process of updating the extension along with the review in the stores.
+
+- **Filtering log**
+
+  ![Filtering log *border](https://cdn.adtidy.org/content/blog/new/log.png)
```

Comment:
- [ ] screenshot should be updated when we will have stable version of filtering log